docs/tutorial.rst

```diff
@@ -1911,7 +1914,7 @@ essential to the network:
 
 .. code-block:: shell
```

Comment:
The output is sorted when the command is run, but is not sorted in the output pasted below.